_data/changelogs/component-character-count.yml

```diff
@@ -2,6 +2,13 @@ title: Character count
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced visual cue when maxlength is exceeded.
+    summaryAdditional:  Now, the visual state of the character count component will reflect its validation state.
```

Comment:
**suggestion:** Recommend that we specifically refer to the error state here for clarity. Here is a possible suggestion (please feel free to re-write):

```suggestion
    summary: Enhanced visual cue when `maxlength` is exceeded.
    summaryAdditional:  Now, the component uses standard USWDS error styles to visually enhance the error state.
```